src/transformers/pipelines/base.py

```diff
@@ -1072,6 +1072,14 @@ def __call__(self, inputs, *args, num_workers=None, batch_size=None, **kwargs):
             )
         elif is_iterable:
             return self.iterate(inputs, preprocess_params, forward_params, postprocess_params)
+        elif self.framework == "pt" and isinstance(self, ChunkPipeline):
```

Comment:
I'm not a huge fan of this condition (mother class referencing subclass).

But it is explicit and quite a minimal code change, that's why I went for it. If anyone has better suggestions I'm taking them

(Duplicate since code modification made it disappear from here)

Reply:
Works for me!